src/ignitus-Routes/ignitus-PublicRoutes/index.tsx

```diff
@@ -45,6 +48,7 @@ export const PublicRoutes: React.FunctionComponent = () => {
             component={AuthenticationRoutes}
           />
           <Route path="/profile" component={StudentProfile} />
+          <Route path="/careersPage" component={CareersPage}/>
```

Comment:
Add this route in the footer for the `Career` as it would redirect to this page.

Reply:
okay